cdflow_commands/account.py

```diff
@@ -8,28 +11,50 @@ def __init__(self, alias, id, role):
 
 class AccountScheme:
 
+    DEFAULT_ENV_KEY = '*'
+
     def __init__(
-        self, accounts, release_account, release_bucket,  default_region
+        self, accounts, release_account, release_bucket,
+        default_region, environment_mapping
     ):
         self.accounts = accounts
         self.release_account = release_account
         self.release_bucket = release_bucket
         self.default_region = default_region
+        self._environment_mapping = environment_mapping
 
-    def create(raw_scheme):
+    @classmethod
```

Comment:
Yeah, I preferred it over hardcoding `AccoutScheme.some_property` in the method.